lib/view_helpers.py

```diff
@@ -11,11 +11,10 @@ def get_after(view, char_limit):
     end = min(view.size(), loc + char_limit)
     return view.substr(sublime.Region(loc, end)), end == view.size()
 
-def is_json_end_line(view, current_location):
+def should_stop_completion_after_end_line(view, current_location):
     last_character = view.substr(max(current_location - 1, 0))
     end_of_line = view.line(current_location).end()
-    is_json_end_line = end_of_line == current_location and last_character == ","
-    return is_json_end_line
+    return end_of_line == current_location and last_character in ",;:"
```

Comment:
extract ` ",;:"` to a const to give it a name

Reply:
plus this is very buggy (think of `lambda x: ` that will not receive suggestions)

Reply:
yes, you will not get completions right after the `:` only after a whitespace